iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -23,6 +23,12 @@
  */
 public final class ValidationUtil {
 
+    public enum ActionOnError {
+        THROW_EXCEPTION,
+        LOG_ERROR,
+        SILENT,
```

Comment:
I think it would be better to rename this option to use a verb instead, since the enum describe an "action".
We could use for instance, `IGNORE`.